### ISSUE TITLE
Embedded videos play audio without clicking, and without focus

### ISSUE THREAD

**Issue by donuts-are-good** [vector-im/element-web#26295]
### Steps to reproduce

1. Where are you starting? What can you see?
a friend posts a video in chat. that video plays with audio. switching to a different room or opening a new element tab later in the day makes this audio play no matter what room im in.
3. What do you click?
nothing.
5. More steps…

#### What did you expect?
a quiet peaceful chat experience

#### What happened instead?
noise

### Operating system

Linux, MacOS, Windows 

### Browser information

Firefox 118.0.1 (64-bit)

### URL for webapp

app.element.io

### Application version

Element version: 1.11.45 Olm version: 3.2.14

### Homeserver

matrix.org

### Will you send logs?

Yes

**Comment by donuts-are-good** [comment]
This has been happening for about a year. Changing "Will you send logs" to no, because it looks like I can't review what info is in those logs before submitting. Also assuming some brave nerd's going to show up saying they can't replicate it too, so if that's the case just close this issue.

**Comment by donuts-are-good** [comment]
Yes it was enabled, I will test with it disabled. However, an autoplaying video should not remain playing when the room is changed, especially since changing back to the video shows it not playing, and it cannot be muted or stopped as a result.

**Comment by t3chguy** [comment]
That sounds like a browser or maybe React bug, we un-render the video element (the whole room view actually) when switching rooms, so it'd be up to your browser to stop the playback.
